laravel/app/Console/Commands/PushSchedule.php

```diff
@@ -74,11 +205,9 @@ public static function updateAssignments($date)
             $objectives = Option::where('id', $option_id)->value('objectives');
             $pref_anest_id = $assignment['anesthesiologist_id'];
             if ($pref_anest_id != null) {
-                $anest = Anesthesiologist::where('id', $pref_anest_id);
-                $fname = $anest->value('first_name');
-                $lname = $anest->value('last_name');
-                $pref_anest_name = "$fname $lname";
-                $pref_anest_staff_key = $anest->value('staff_key');
+                $anesthesiologist = Anesthesiologist::find($pref_anest_id);
+                $pref_anest_name = "$anesthesiologist->first_name $anesthesiologist->last_name";
```

Comment:
Have you done manual testing to make sure this string interpolation will work as expected?